images/node-feature-discovery/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+name: node-feature-directory
+image: cgr.dev/chainguard/node-feature-directory
+logo: https://storage.googleapis.com/chainguard-academy/logos/grafana.svg
+endoflife: ""
+console_summary: ""
+short_description: A minimal wolfi-based image for node-feature-discovery, Node feature discovery for Kubernetes
+compatibility_notes: ""
+readme_file: README.md
+upstream_url: https://github.com/grafana/https://github.com/kubernetes-sigs/node-feature-discovery
+keywords:
+  - kubernetes
+  - application
+  - node-discovery
```

Comment:
I'm not sure but you cannot add an entry like this `node-discovery`, but again, I'm not sure.

Reply:
I've changed the tag to "hardware" since it adds hardware-related tags to the k8s nodes